projects/rero/ng-core/src/lib/record/editor/editor.component.ts

```diff
@@ -913,4 +916,14 @@ export class EditorComponent implements OnInit, OnChanges, OnDestroy {
       };
     }
   }
+
+  /**
+   * Handle form error
+   * @return Observable<Error>
+   */
+  private _handleError(error: { status: BigInteger, title: string }): Observable<Error> {
```

Comment:
Why `BigInteger`, this cannot be just `number`?

Reply:
done